rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/RMServiceImpl.java

```diff
@@ -365,53 +372,59 @@ public String getMBeanInfo(String sessionId, String name, List<String> attrs)
     @Override
     public String getNodeMBeanInfo(String sessionId, String nodeJmxUrl, String objectName, List<String> attrs)
             throws RestServerException, ServiceException {
-        return executeFunctionReturnStreamAsString(restClient -> restClient.getNodeMBeanInfo(sessionId,
-                                                                                             nodeJmxUrl,
-                                                                                             objectName,
-                                                                                             attrs));
+        return executeFunctionReturnStreamAsStringWithoutNewLines(restClient -> restClient.getNodeMBeanInfo(sessionId,
+                                                                                                            nodeJmxUrl,
+                                                                                                            objectName,
+                                                                                                            attrs));
     }
 
     @Override
     public String getNodeMBeanHistory(String sessionId, String nodeJmxUrl, String objectName, List<String> attrs,
             String timeRange) throws RestServerException, ServiceException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)